src/main/java/reposense/authorship/FileInfoExtractor.java

```diff
@@ -91,7 +105,7 @@
         }
 
         String[] fileDiffResultList = fullDiffResult.split(DIFF_FILE_CHUNK_SEPARATOR);
-        Set<Path> nonBinaryFilesSet = getNonBinaryFilesList(config);
+        Set<Path> nonBinaryFilesSet = getFilesList(config, false);
```

Comment:
Perhaps can have a `getNonBinaryFilesSet` and `getBinaryFilesSet` that calls `getFilesList`? it will be more clearer

Reply:
I don't think it is worth it to have such a thin wrapper over the function, considering that we should change `getAllFileInfo` to `getAllNonBinaryFileInfo` and `getAllBinaryFileInfo` as well. 
That is 4 new methods and possibly even more methods for the same concept of indicating if binary or non-binary files is requested, which would add more complexity to the code.

However, I can see that the variable name is somewhat confusing. How about changing `isBinaryFiles` to `binaryFilesWanted` or `binaryFilesRequested`?